### ISSUE TITLE
Spinner running only full screen

### ISSUE THREAD

**Issue by xereda** [yuche/vue-strap#336]
The spinner component is not adjusting as property ": fixed". It's always getting in full screen.

**Comment by wffranco** [comment]
The `fixed` attribute must be a boolean (using `:fixed="true/false"` or `fixed="true/false`), or just put/omit the attribute (not assigning any value) if you want it fixed/normal.

**Comment by SeoRoman** [comment]
I have to agree... I have removed fixed, tried :fixed="false" and fixed="false" nothing resolves the full screen issue... is there any docs on how this should be implemented into the HTML?
